src/python/pants/backend/python/lint/black/lockfile.txt

```diff
@@ -111,7 +111,7 @@ typed-ast==1.4.3; python_version < "3.8" and python_full_version >= "3.6.2" \
     --hash=sha256:209596a4ec71d990d71d5e0d312ac935d86930e6eecff6ccc7007fe54d703808 \
     --hash=sha256:9c6d1a54552b5330bc657b7ef0eae25d00ba7ffe85d9ea8ae6540d2197a3788c \
     --hash=sha256:fb1bbeac803adea29cedd70781399c99138358c26d05fcbd23c13016b7f5ec65
-typing-extensions==3.10.0.2; python_version < "3.8" and python_full_version >= "3.6.2" and python_version >= "3.6" \
+typing-extensions==3.10.0.2 \
```

Comment:
The previous env markers were more correct iiuc..but it looks like this change doesn't break anything. (We test that the default lockfile works for all Python interpreters and macOS x Linux). So, this seems fine and of course we should not manually rewrite the generated lockfile.